FlowCrypt/LaunchFlow/FlowController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FlowController.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 07/02/2020.
+//  Copyright © 2020 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+protocol FlowController: FlowStepHandler { }
+
+final class DefaultFlowController: FlowController {
```

Comment:
Better call all of these `DefaultLaunchFlowController`, `LaunchFlowController` and similar. Add the `Launch`, else it's unclear what it does.

Reply:
DefaultFlowController is responsible for executing LaunchFlow steps.
Renamed just to `FlowController` 

`LaunchFlowController` starts flow with some launch context.